### PR TITLE
added dropdown headers and fixed window.location.replace

### PR DESCRIPTION
**Pull request by Andysu7117**
Also changed  the location paths for images

### CODE REVIEW

File: display-search.html [hunk 1]
```diff
@@ -1,74 +1,117 @@
...
+                </h1>
+              </li>
+            </ul>
+          </div>
+          <div class="top-bar-middle small-4">
+            <ul class="dropdown menu" data-dropdown-menu>
+              <li>
+                <a href="#">Movies</a>
+                <ul class="menu">
+                  <li><a href="#">Popular</a></li>
```
Comment (shimna-puthanayil) [inline display-search.html]: Change the popular to top rated

**Review by shimna-puthanayil — CHANGES_REQUESTED** [review]
Make changes on all lines i have commented

File: display-search.html (hunk 1 above)
Comment (Andysu7117) [reply display-search.html]: Changed to Top Rated

**Review by shimna-puthanayil — APPROVED** [review]
Approved

**Pull request merged**